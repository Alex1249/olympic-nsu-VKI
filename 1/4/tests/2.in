1000000 55
0
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
999983
999984
999985
999986
999987
999988
999989
999990
999991
999992
999993
999994
999995
999996
999997
999998
999999
997
994009
994008
994010
996004
65536
65535
65537
