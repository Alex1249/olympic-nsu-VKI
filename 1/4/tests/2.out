0 not
1 not
2 prime
3 prime
4 not
5 prime
6 not
7 prime
8 not
9 not
10 not
11 prime
12 not
13 prime
14 not
15 not
16 not
17 prime
18 not
19 prime
20 not
21 not
22 not
23 prime
24 not
25 not
26 not
27 not
28 not
29 prime
999983 prime
999984 not
999985 not
999986 not
999987 not
999988 not
999989 not
999990 not
999991 not
999992 not
999993 not
999994 not
999995 not
999996 not
999997 not
999998 not
999999 not
997 prime
994009 not
994008 not
994010 not
996004 not
65536 not
65535 not
65537 prime
